topics/reference_guide/custom_language_support/websymbols.md

```diff
@@ -0,0 +1,31 @@
+<!-- Copyright 2000-2023 JetBrains s.r.o. and other contributors. Use of this source code is governed by the Apache 2.0 license that can be found in the LICENSE file. -->
+# Web Symbols
+
+<link-summary>
+Web Symbols framework simplifies web technology development by utilizing Symbol API and supporting custom syntaxes.
+</link-summary>
+
+> This API is available starting from 2022.3.1 and is currently in development and thus in an experimental state.
+>
+{style="warning"}
+
+Web Symbols is a framework built on top of the platform's [Symbol API](symbols.md).
+Its primary use is to reduce boilerplate when building support for web technologies,
+however one can take advantage of it for any kind of technology.
+Web Symbols API reduces work needed to provide reference resolution, find usages, documentation and rename refactoring.
```

Comment:
```suggestion

Web Symbols API reduces work needed to provide reference resolution, find usages, documentation and rename refactoring.
```
I would add a line break here (also in other places), to allow taking a breath every 2-3 sentences :)